code/controllers/subsystem/mapping.dm

```diff
@@ -237,6 +237,27 @@ SUBSYSTEM_DEF(mapping)
 		INIT_ANNOUNCE("Loaded [name] in [(REALTIMEOFDAY - start_time)/10]s!")
 	return parsed_maps
 
+/datum/controller/subsystem/mapping/proc/LoadStationRooms()
+	var/start_time = REALTIMEOFDAY
+	for (var/obj/effect/spawner/room/R in world)
```

Comment:
Just track the spawners in a list

Reply:
cannot track them since everything isn’t inited at this point unless it’s added to the list on New()